traits/src/resource.rs

```diff
@@ -110,6 +107,20 @@ pub struct ResourceInfo<BoundedString, BoundedParts> {
 	pub pending_removal: bool,
 }
 
+#[derive(Encode, Decode, Eq, PartialEq, Clone, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+#[cfg_attr(feature = "std", derive(Serialize))]
+#[cfg_attr(
+	feature = "std",
+	serde(bound = r#"
+			BoundedString: AsRef<[u8]>,
+			BoundedParts: AsRef<[PartId]>
+		"#)
+)]
+pub struct ResourceInfoMin<BoundedString, BoundedParts> {
```

Comment:
What does `Min` in `ResourceInfoMin` stands for?

Reply:
Open for better naming suggestions.
 ResourceInfo - contains extra props, ResourceInfoMin - 'minimal', has only `id`. 
Another option was  `ResourceWithid`. wdyt?

Reply:
What is the case for using `ResourceInfo` vs `ResourceInfoMin`?

Reply:
Use case - adding resources with explicit id during minting. 
https://github.com/rmrk-team/rmrk-substrate/blob/d5c290d3053ebdc289990444b42aa40f3baf8daa/pallets/rmrk-core/src/functions.rs#L344

![image](https://user-images.githubusercontent.com/1027871/183288597-a8756d04-b2b0-44b4-b8d1-868d938376d8.png)

So it's similar to https://github.com/rmrk-team/rmrk-substrate/blob/d5c290d3053ebdc289990444b42aa40f3baf8daa/traits/src/resource.rs#L96-L108

Just without `pending_removal` and `pending` fields.

Reply:
I think the old naming `ResourceInfoWithId` was reasonable, but the proposed `ResourceInfoMin` should work. So whichever one is okay with me.